doc/plotting/index.rst

```diff
@@ -362,5 +366,13 @@ NOTE: These functions works for only with matplotlib higher than 1.3.1.
                          |hack|
                          **Example:**
                          :ref:`sphx_glr_auto_examples_01_plotting_plot_surf_stat_map.py`
+
+|niimg_to_surf_data|     :func:`niimg_to_surf_data`
+                         |hack|
+                         Projecting a 3D brain image onto the cortical surface.
```

Comment:
I am just trying to understand. "Projecting a 3D statistical map onto" ........ if this function is more suitable to plot statistical maps. 3D brain image seems like a more general or averaged one across timeseries of images.